a10_ansible/axapi_http.py

```diff
@@ -300,21 +300,36 @@ def delete(self, url, params={}, **kwargs):
 
     def activate_partition(self, partition):
         url = "/axapi/v3/active-partition"
-        payload = { 
+        payload = {
             "curr_part_name": partition["name"],
             "shared": partition["shared"]
         }
         try:
             self.post(url, {"active-partition": payload})
         except Exception as ex:
-            raise Exception("Could not activate due to: {0}".format(ex)) 
+            raise Exception("Could not activate due to: {0}".format(ex))
+
+    def change_context(self, device_context_id):
+        url = "/axapi/v3/device-context"
+        payload = {
+            "device-context": {
+                "device-id": device_context_id
+            }
+        }
+        try:
+            self.post(url, payload)
+        except Exception as ex:
```

Comment:
AXAPI related exceptions should be caught using https://github.com/a10networks/a10-ansible/blob/master/a10_ansible/errors.py#L16 or some class inherited from that base one.

We have code that raises the error itself here: https://github.com/a10networks/a10-ansible/pull/129/files#diff-c3cb3ae0871d4dbbd38bce30dfe29240R171 in the same file